lib/httplog/http_log.rb

```diff
@@ -95,8 +95,12 @@ def log_body(body, encoding = nil, content_type=nil)
 
     end
 
-    def log_data(data)
+    def log_data(data, content_type = nil)
       return if config.compact_log || !config.log_data
+      unless text_based?(content_type)
```

Comment:
This is breaking all other adapters, since they're not passing in the request content type, and the default `nil` will not be treated as text based.